RELEASES.md

```diff
@@ -72,12 +72,9 @@ Only the following major release series have a stable release status:
 |`v2.2.x`|February 01, 2023|
 |`v2.3.x`|February 01, 2023|
 |`v2.4.x`|February 01, 2023|
-|`v3.0.x`|March 15, 2023|
-|`v3.1.x`|March 15, 2023|
```

Comment:
I wonder if we should note that these have been discontinued? Maybe a blanket statement like all missing minor release versions have been discontinued